drop-in/src/main/java/com/adyen/checkout/dropin/DropInConfiguration.kt

```diff
@@ -376,6 +379,11 @@ class DropInConfiguration private constructor(
             return this
         }
 
+        fun overridePaymentMethodName(paymentMethodType: String, name: String): Builder {
```

Comment:
This makes it easy for merchants to override payment method name. With design changes, we might introduce a description field for payment methods.
I decided to keep it as simple as possible for now, but later we can also offer a method, which will require `DropInPaymentMethodInformation`, instead of `name` string.

Reply:
Good idea 👍 Let's make sure we document the method and mention that it can be called multiple times, for different payment methods

Reply:
A few things on the documentation, all minor comments/nitpicking:
- We generally refer to it as "Drop-in" instead of "drop in" in other code docs as well as the online docs.
- The wording of "filtering" might be confusing since we're not filtering the list itself, but not sure about this one.
- The "Only the latest value will be shown..." line can be removed since it describes a scenario caused by a wrong integration from merchants, it's good that we handle it internally but there's no need to document it if they shouldn't use it this way.
- With most docs you already explain all the logic at the start, so personally I prefer to keep the `@param`s section very short and only describe what the parameter is. For example, paymentMethodType: the type of the `PaymentMethod` and name: the name to be displayed.